exercise/cache/content.py

```diff
@@ -9,6 +9,7 @@
 from lib.cache import CachedAbstract
 from ..models import LearningObject, BaseExercise
 from .hierarchy import ContentMixin
+from django.conf import settings
```

Comment:
coding style: import statements are grouped
1) Python builtin modules
2) 3rd-party modules like Django
3) the project's internal modules